tests/task/test_graph_task.py

```diff
@@ -249,15 +249,14 @@ def test_pre_process_input_conversation_collection(
         }
 
     def test_pre_process_input_data_group_dependent_fields(self):
-        """Test processing inputs where fields have been marked as dependent"""
+        """Test processing inputs where fields have been marked as dependent
+        Identity data has also been marked as being dependent.
```

Comment:
Should this read `Test processing inputs where identity data has also been marked as being dependent.`?

Reply:
Well it's testing both, that reference fields and an identity field have all been marked together as a group, I'll rephrase!